### ISSUE TITLE
I have no idea what I'm doing

### ISSUE THREAD

**Issue by rkansawyer** [reworkd/AgentGPT#268]
I've been running AgentGPT with a paid API key from OpenAI and have no idea how to use it successfully. All it does is tell me that it has completed tasks when it hasn't.

I've run several scenarios with various goals; each time it tells me it has done N number of things in pursuit of that goal but there is no evidence that has done anything at all. For example, it claimed that it had successfully gotten one of my Twitter accounts to 10k followers (it has not), and that it had successfully built a client list for my business (it didn't). I have never seen any of the content it claims to have created and circulated on the internet for me. Virtually none of the information it has returned has been accurate.

what the hell is going on

**Comment by peti123x** [comment]
I'm in the same boat. I have given it a github user & token and told it to create an empty repository and publish it using the credentials. It understands it should use them to authentication and it even gives me a fake repository URL claiming it has published it, but, it just does nothing. am I expecting too much?

**Comment by rkansawyer** [comment]
I don't know if we're expecting too much for baseline functionality; so far for me its miss rate is 100% and its consistently misrepresenting itself about what it is doing. A lot of people are raving about this tech but I don't see any evidence that it works at all on my end. I'm confused.

**Comment by peti123x** [comment]
Yeah i'm exactly the same. There seems to be a few other threads where people say the same thing as you and me that it doesnt do anything but it seems to be next to nothing compared to how many people are amazed by what it can do, so I assume we must be doing something wrong? I mean I would expect it to be able to create a repository and push it with credentials given at the very least, nevermind everything else I expected it to do, but so far based on what I've seen all it does is interpret what it needs to do and "does it in theory" (aka describes the task back and what it "did") but then actually does not achieve it.. Im confused, too.

**Comment by peti123x** [comment]
Would be nice to see an answer from someone, I'm not sure if its just because I'm trying with a paid GPT3 license and not GPT4 (the one you have to sign up for and get invited to), but if so that's pretty silly and should be quite obviously displayed somewhere. Otherwise I'm out of ideas really, just seems to do nothing of material.

**Comment by rkansawyer** [comment]
it's particularly irritating given that the machine keeps telling me that it has accomplished all of these miraculous things, but when I search for them there is nothing. Am I still in demo mode or something? is it even interacting with the internet? it's claiming to have done all of these things with incredible speed, too, which seems implausible even for a computer. Between that and the inaccuracy in the information it provides back, I can't imagine a scenario where this is useful for literally _anything_.

So surely we're missing something. Surely it's not the case that everyone is stirring up hype about a "revolutionary" technology that can't do half a percent of what it's proponents claim.

**Comment by peti123x** [comment]
What I find particularly interesting is the difference of output when I input the same goal on https://agentgpt.reworkd.ai/ vs my local version (bearing in mind I also have my paid key input locally):
"Create a 'Hello World' Laravel project using the latest stable version"
On the live deployment, this will (at least) make it look like it's actually processing things; it creates several tasks for doing this (e.g. downloading dependencies, installation, post installation steps, database configuration) and then it times out saying that's all it can do without a key. Fine. Then inputting the same goal locally will "finish" almost immediately telling me that it's created a new project, with no evidence of it has achieveing that. And even more so, scaffolding a project with Laravel being only the subgoal of a task, it just seems to glaze over the installation and just immediately tell me "yup ok done that" when evidently it hasnt, even just judging by the processing time.

I don't get it. I almost want to try more of the projects (particularly the ones that have near enough a 100k stars), since _surely_ people are not starring and raving about this when it achieves nothing? But then people say this one works, too, so I'm just really unsure what's happening here lol.
